catalog/core/catalog-core-versioning/versioning-common/src/main/java/ddf/catalog/core/versioning/impl/MetacardVersionImpl.java

```diff
@@ -183,44 +183,77 @@ public MetacardVersionImpl(String id, Metacard sourceMetacard, Action action, Su
         this.setTags(Collections.singleton(VERSION_TAG));
     }
 
+    //@formatter:off
```

Comment:
Our formatter and checkstyle break on this case (it gets formatted incorrectly and fails checkstyle). It will necessitate fixing our checkstyle (though I think there are larger debates to be had regarding this)